x/distribution/types/store.go

```diff
@@ -1,7 +1,26 @@
 package types
 
-const ()
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
 
-// store: per-validator deccoins, for bonded set
-// store: per-validator accumulated commission
-// store: storage records from withdrawals (fees / power)
+// historical rewards for a validator
+type ValidatorHistoricalRewards struct {
```

Comment:
Can this also have an int8 or int16 refcount, so in the future its easier to add the state pruning? (Or would it be better to add the pruning all at once)

Reply:
Was planning to add in separate PR if we decide to go with F1.

Reply:
Added a TODO and ref link for now.